packages/fast-components-react-msft/src/badge/badge.props.ts

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import {
+    BadgeClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+
+export enum BadgeAppearance {
+    accent = "accent",
+    highlight = "highlight",
+    lowlight = "lowlight",
+    lightweight = "lightweight",
```

Comment:
@nalogiudice I don't see lightweight being used or included anywhere else - can we remove this?

Reply:
It is, in fact, being used. When switching in the editor, lightweight is the non-backplate badge. Instead of adding a class, it is just plain `badge`. The other backplate classes then get applied.

Reply:
Lightweight removed